lib/typings/common-types.ts

```diff
@@ -61,7 +61,7 @@ export function assertSingleKey(
 /**
  * Typing wrapper around Object.keys()
  */
-export function objectKeys<T>(object: T) {
+export function objectKeys<T extends {}>(object: T) {
```

Comment:
I could've extended from `{}` or `object` here. I had no preference, happy to change as needed.